src/diff/index.js

```diff
@@ -378,8 +393,21 @@ function catchErrorInComponent(error, component) {
 			}
 			catch (e) {
 				error = e;
+				isSuspend = typeof error.then === 'function';
+				suspendingComponent = component;
 			}
 		}
 	}
+
+	if (isSuspend && suspendingComponent) {
+		// TODO: what is the preact way to determine the component name?
+		const componentName = suspendingComponent.displayName || (suspendingComponent._vnode && suspendingComponent._vnode.type
```

Comment:
Maybe we can move this section into `preact/debug`. A `Suspense` component without a fallback doesn't seem to make much sense, so we could just always warn on that. This would also keep core small and tiny :tada:

Reply:
This also allows the recheck of isSuspsend and suspendingComponent, so that will save a lot

Reply:
I still kept the `Missing Suspense` error here and throwing it on the `suspendingComponent`.

Would you want to save some bytes and just throw the error at the end of the function instead of the `suspendingComponent`?

Reply:
What he's saying is to throw it in a hook from preact/debug since it doesn't really matter in production anyway

Reply:
Can you guys give me a hint on where I could add this to `preact/debug`. I did not find a proper place to add it